liwords-ui/src/admin/tourney_editor.tsx

```diff
@@ -121,17 +130,20 @@ const SettingsModalForm = (mprops: {
 };
 
 export const TourneyEditor = (props: Props) => {
-  const [description, setDescription] = useState("");
```

Comment:
This code was kinda an anti-pattern IMO. For react forms, you have a choice between a "controlled" form where you manage all of the `onChange` events, set the state locally to your component, and then bind the `value` of that input to the state. _or_ you can have an uncontrolled component where the state is managed externally

Here we were doing both, so there were essentially two sources of truth for field values: one in the local state of this component and one within the `Form.useForm()` hook state. I've removed the controlled state entirely, it shouldn't affect any behavior and just kills some dead code

Reply:
thanks for fixing that; i am aware of the controlled vs uncontrolled but I must have totally brainfarted.